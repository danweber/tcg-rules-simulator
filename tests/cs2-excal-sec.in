VERSION:2
FIELD1:Attacker,DW1-Van Mon1,CS1-03 Mon2,CS1-03 Gideon4,CS1-17
SECURITY2:CS2-Excalibur CS2-Excalibur
GAME:P1 T1 MAIN 7
DUMP:SECURITY,TRASH


ECHO: -- attack into sec
CMD:ATTACK Attacker Player
STEP:AUTOANSWER:CAPTURE
DUMP:SECURITY,TRASH

EXIT
