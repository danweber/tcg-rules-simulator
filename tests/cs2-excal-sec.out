FIELD1:Attacker,DW1-Vanillamon Mon1,CS1-03 Mon2,CS1-03 Gideon4,CS1-17
FIELD2:
SECURITY1:
SECURITY2:CS2-Excalibur CS2-Excalibur
TRASH1:
TRASH2:
GAME:P1 T1 MAIN 7
ECHO: -- attack into sec
CAPTURE:Target Garbage Can Lid 2
CAPTURE:Target Garbage Can Lid 3
FIELD1:Mon2,CS1-03 Gideon4,CS1-17
FIELD2:
SECURITY1:
SECURITY2:CS2-Excalibur
TRASH1:CS1-03 DW1-Vanillamon
TRASH2:CS2-Excalibur
GAME:P1 T1 MAIN 7
